.github/workflows/docker-publish.yml

```
@@ -11,10 +11,14 @@ on:
 permissions:
   contents: read
   packages: write
+  secrets: read
```

actionlint (1.7.7)

unknown permission scope "secrets". all available permission scopes are "actions", "attestations", "checks", "contents", "deployments", "discussions", "id-token", "issues", "packages", "pages", "pull-requests", "repository-projects", "security-events", "statuses"

(permissions)